L1Trigger/L1TGlobal/plugins/L1TGlobalPrescaler.cc

```diff
@@ -100,18 +99,38 @@ namespace {
 #include "FWCore/Framework/interface/one/EDFilter.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
 #include "FWCore/Utilities/interface/EDMException.h"
 #include "CondFormats/DataRecord/interface/L1TGlobalPrescalesVetosRcd.h"
 #include "CondFormats/L1TObjects/interface/L1TGlobalPrescalesVetos.h"
 #include "DataFormats/L1TGlobal/interface/GlobalAlgBlk.h"
 
+
+namespace {
+
+  template <typename T, std::size_t N, typename S>
```

Comment:
@Dr15Jones @wddgit can you think of any way to support
```c++
template <typename T, std::size_t N>
std::array<T, N>
ParameterSet::getParameter<std::array<T, N>>(...)
```
directly in `ParameterSet` ?

I I understand that it cannot be done as an overload or partial specialisation of `ParameterSet::getParameter<T>(...)` .